browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -1522,10 +1522,9 @@ void RewardsDOMHandler::OnFetchBalance(
     base::DictionaryValue data;
     data.SetIntKey("status", result);
     base::Value balance_value(base::Value::Type::DICTIONARY);
+    balance_value.SetDoubleKey("total", balance->total);
```

Comment:
Since `balance` is always valid, do we need the `&& balance` check in the `if` below?

Reply:
that was my mistake as balance can be null, so we should still have checks for it. Fixed